bin/lisky

```diff
@@ -14,25 +14,26 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-var os = require('os');
-var lockfile = require('lockfile');
-var semver = require('semver');
+const os = require('os');
+const lockfile = require('lockfile');
+const semver = require('semver');
+const packageJSON = require('../package.json');
 
-var packageJSON = require('../package.json');
+// eslint-disable-next-line import/extensions,import/no-unresolved
+const lisky = require('../dist').default;
```

Comment:
We don't want to do this yet because it will initialize everything even if the user only wants to know the version or clean the config lockfile.

Reply:
Perhaps going forward we should expose a callable function from `dist` and call it here, rather than calling it there already.